IdentityCore/tests/automation/ui_app_lib/ui_common/ios/MSIDAutomationRequestViewController.m

```diff
@@ -1,62 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// All rights reserved.
-//
-// This code is licensed under the MIT License.
-//
-// Permission is hereby granted, free of charge, to any person obtaining a copy
-// of this software and associated documentation files(the "Software"), to deal
-// in the Software without restriction, including without limitation the rights
-// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
-// copies of the Software, and to permit persons to whom the Software is
-// furnished to do so, subject to the following conditions :
-//
-// The above copyright notice and this permission notice shall be included in
-// all copies or substantial portions of the Software.
-//
-// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-// THE SOFTWARE.
-
-#import "MSIDAutomationRequestViewController.h"
-#import "MSIDAutomationTestRequest.h"
-
-@interface MSIDAutomationRequestViewController ()
```

Comment:
Maybe we can support both modes in the automation library:
1. Files based (default)
2. UI based - used for tests that are flaky or don't work with a files based model

Do you see any benefit in supporting both of those models?

Reply:
I don't see the benefit in UI bases pipeline, files communication is faster and has less code.

Reply:
Added support of ui pipeline for the real device.